a3p-integration/proposals/z:acceptance/test-lib/index.js

```diff
@@ -20,3 +21,14 @@ export const walletUtils = await makeWalletUtils(
   { delay, execFileSync, fetch },
   networkConfig,
 );
+
+// eslint-disable-next-line @jessie.js/safe-await-separator -- buggy version
+export const governanceDriver = await makeGovernanceDriver(
+  fetch,
+  networkConfig,
+);
```

Comment:
Injecting `fetch` and `networkConfig` should be left to test scripts and not in a module like this.

see [OCap Discipline](https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline)

Reply:
updated. 
also isnt the `walletUtils` object in this file breaking this principle?

Reply:
yes.